### PR TITLE
feat(link): adds default link style

### PR DESCRIPTION
**Pull request by codemonkey800**
Adds default link style

---
title: "type_of_change(scope_of_work): ticket title"
reviewers: "sds-eng, sds-design"
---

## Summary

Shortcut ticket: [sh-154877](https://app.shortcut.com/sci-design-system/story/154877/update-default-link-component)
Update Default Link Component

## Checklist

- [ ] Default Story in Storybook
- [ ] LivePreview Story in Storybook
- [ ] Test Story in Storybook
- [ ] Tests written
- [ ] Variables from `defaultTheme.ts` used wherever possible
- [ ] If updating an existing component, depreciate flag has been used where necessary
- [ ] Chromatic build verified by @sds-design

### CODE REVIEW

File: src/core/Link/index.tsx [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
-  return { root: { "&:hover": { color: theme?.app?.colors.primary[600] } } };
-});
+const Link = (props: LinkProps, ref: ForwardedRef<HTMLAnchorElement>) => {
+  const { sdsStyle } = props;
+  let underline: LinkProps["underline"];
 
-const Link = (props: LinkProps): JSX.Element => {
-  const classes = useStyles();
+  if (sdsStyle === "default") {
+    underline = "none";
```
Comment (tihuan) [inline src/core/Link/index.tsx:9]: niiice paving the way for underline variant already 🔥 !

**Review by tihuan — APPROVED** [review]
Looks fantastic 🎉 !! Thanks again for adding this, Jeremy 🤩 👏 !1

**Review by mdjmoore — APPROVED** [review]
(◕ᴥ◕)

**Comment by tihuan** [comment]
Woohoo! Thanks so much, Connor! Have an amazing weekend 🤩🎈

**Pull request merged**